src/codec_aom.c

```diff
@@ -737,12 +739,12 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
         if (encoder->maxThreads > 1) {
             aom_codec_control(&codec->internal->encoder, AV1E_SET_ROW_MT, 1);
         }
-        if (encoder->tileRowsLog2 != 0) {
-            int tileRowsLog2 = AVIF_CLAMP(encoder->tileRowsLog2, 0, 6);
+        if (tileRowsLog2 != 0) {
+            tileRowsLog2 = AVIF_CLAMP(tileRowsLog2, 0, 6);
```

Comment:
Nit: It can never clamp to 0. Should it be `AVIF_CLAMP(tileRowsLog2, 1, 6)`?

Reply:
Hmm... you are right. I found that `AVIF_CLAMP(tileRowsLog2, 1, 6)` will change the behavior for a negative `tileRowsLog2`. So the current code is correct. It will call `aom_codec_control(AV1E_SET_TILE_ROWS)` with the value 0 unnecessarily (0 is the default value). Since a negative `tileRowsLog2` is uncommon, I think this is fine.

The alternative is the following, but we will always do the `AVIF_CLAMP()`:

```
        tileRowsLog2 = AVIF_CLAMP(tileRowsLog2, 0, 6);
        if (tileRowsLog2 != 0) {
            aom_codec_control(&codec->internal->encoder, AV1E_SET_TILE_ROWS, tileRowsLog2);
        }
```

Would you prefer that? If so, I can make the change in a follow-up.

Reply:
It does not matter much, it can be left as is.
Alternatively an error could be output when tileRowsLog2 < 0.